LayoutTests/animations/animation-delay-changed.html

```diff
@@ -47,7 +47,7 @@
 
       const expectedValues = [
         // [animation-name, time, element-id, property, expected-value, tolerance]
-        [null, 1.0, "box1", "left",  200, 10],
```

Comment:
Would the value always be `200` if we just added `-webkit-animation-fill-mode: forwards`?

It seems like this test is checking if it ended but the behavior is if the check is done at the end its already snapped back to `0`.

Reply:
Yes, adding `-webkit-animation-fill-mode:forwards` also resolves the problem.
But, I think my approach is better than yours.
This test is checking if box1 is exactly 0.5s earlier than box2. With your approach, box1 can be more earlier.